x-pack/plugins/cloud_security_posture/server/create_transforms/create_transforms.ts

```diff
@@ -7,13 +7,16 @@
 import { transformError } from '@kbn/securitysolution-es-utils';
 import { TransformPutTransformRequest } from '@elastic/elasticsearch/lib/api/types';
 import type { ElasticsearchClient, Logger } from '@kbn/core/server';
+import { errors } from '@elastic/elasticsearch';
 import { latestFindingsTransform } from './latest_findings_transform';
 
 // TODO: Move transforms to integration package
 export const initializeCspTransforms = async (
   esClient: ElasticsearchClient,
   logger: Logger
 ): Promise<void> => {
+  const LATEST_TRANSFORM_V830 = 'cloud_security_posture.findings_latest-default-0.0.1';
```

Comment:
we should be able to get this constant from somewhere? would aid in clarifying why it's used too

Reply:
it is the previous naming of the transform - that is how I got the name (check out the previous implementation)